### DISCUSSION TITLE
Como submeter sua palestra para um evento da perifaCode

### DISCUSSION THREAD

**Discussion by gitlherme** [perifacode/comunidade#21] [Anúncios]
* [1. Abra uma issue no repositório da comunidade (este repositório)](#Abraumaissuenorepositriodacomunidadeesterepositrio)
	* [Como abrir uma issue durante um call for papers?](#Comoabrirumaissueduranteumcallforpapers)
* [2. Preenchendo os campos da sua palestra](#Preenchendooscamposdasuapalestra)
* [3. Como saber se a minha palestra foi aceita?](#Comosaberseaminhapalestrafoiaceita)

Dentro do perifaCode, queremos que todas as pessoas possam compartilhar suas experiencias e aprendizados.

Para isso, tentamos montar uma forma simples para que possam submeter suas palestras em nossos eventos. 

Cola com nóis e se liga. 👇🏽

## <a name='Abraumaissuenorepositriodacomunidadeesterepositrio'></a>1. Abra uma issue no repositório da comunidade (este repositório)

Criamos um issue template com algumas informações necessárias para a comunidade avaliar e te votar em sua apresentação para os nossos eventos.

> Mas o que é um issue template?

É um modelo padrão para um possível "problema" que deve ser resolvido, conhecido como issues aqui dentro do GitHub. Como a gente é de quebrada e sempre dá nosso jeitinho, aqui vamos usar ela como um modelo padrão de call for papers (chamada para apresentações), para manter tudo organizado e padronizado em nossos eventos.

Você pode saber mais sobre issue template clicando [aqui](https://docs.github.com/pt/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).

### <a name='Comoabrirumaissueduranteumcallforpapers'></a>Como abrir uma issue durante um call for papers?

Dentro do nosso [repositório da comunidade](https://github.com/perifacode/comunidade), vá até **Issues** e depois clique em **New issue** ou [clique aqui](https://github.com/perifacode/comunidade/issues/new/choose) para cair direto na página de escolha do template.

![](https://imgur.com/ooLPyoS.png)
![](https://imgur.com/tlAwvps.png)

Após isso, clique em **Get Started** no template **Call for papers**.

![](https://imgur.com/qFIHslE.png)

Feito isso, voce estará na página de abertura de issue.
![](https://imgur.com/h6oyKgP.png)

## <a name='Preenchendooscamposdasuapalestra'></a>2. Preenchendo os campos da sua palestra

Após entrar na página de issue, preencha todos os campos que estão descritos no modelo para enviar a sua palestra.

No título, mantenha o prefixo [C4P] e após o hífen coloque o título da sua palestra.

Na descrição, comente um pouco sobre o que é a sua palestra, quais os temas que serão abordados, qual o objetivo que vc pretende atingir com ela e o que mais achar necessário.

Dentro de nível técnico, deixe checado o nível técnico da palestra, além do tipo de palestra que ela é.

Após isso, basta clicar em **Submit new issue**.

## <a name='Comosaberseaminhapalestrafoiaceita'></a>3. Como saber se a minha palestra foi aceita?

Tentaremos encaixar todas as palestras enviadas nos nossos eventos, mas claro que algumas vezes, pode ser que fique com uma fila muito grande de palestras, então tentaremos usar um modelo de reação para saber quais são as que o pessoal mais sente necessidade no momento, ou então usaremos as palestras que façam mais sentido para o próximo evento em questão de tema.

Para isso, após submeter sua palestra, clique no ícone de emoji, no canto superior direito da sua issue, e adicione o emoji 👍 como reação para vermos a adoção dos membros da comunidade em relação aquela palestra, assim conseguimos colocar as palestras que os membros mais precisam nos eventos mais próximos.

O resultado esperado é como o da imagem abaixo.

![](https://imgur.com/xY7nUzV.png)

E é isso, muito obrigado por submeter sua palestra! 🎉💛

Caso tenham alguma dúvida, usem o espaço de comentários dessa issue para marcar os organizadores e saná-las.

**Comment by lucasfsousa** [comment]
@gitlherme parabéns pelo post bem detalhado, mas uma coisa que não ficou clara, o evento seria online ou presencial?

**Reply by gitlherme** [reply]
Fala mano, desculpa!

O evento será online, na nossa Twitch! http://twitch.tv/perifacode
